packages/chat-stateful-client/src/StatefulChatClient.ts

```diff
@@ -135,7 +136,7 @@ export const createStatefulChatClient = (
   options?: StatefulChatClientOptions
 ): StatefulChatClient => {
   return createStatefulChatClientWithDeps(
-    new ChatClient(args.endpoint, args.credential, options?.chatClientOptions),
+    new ChatClient(args.endpoint, args.credential, addApplicationId(options?.chatClientOptions)),
```

Comment:
If I wanted to add more options here. I would have to rewrite this function `addApplicationId` or call it inside another.
I am not seeing a lot of value in this mechanism.
Might be better to just explicitly use here.
```
{
    ...options,
    userAgentOptions: { userAgentPrefix: getApplicationId() }
}
```

Am I missing something?

Reply:
No reason. This used to be a fatter function but I extracted that into acs-ui-common.

In any case, thanks to your comment I realized that I should be passing in the modified options in both arguments. Updated.